src/main/resources/com/amazon/jenkins/ec2fleet/EC2FleetCloud/config.jelly

```diff
@@ -4,16 +4,12 @@
          xmlns:t="/lib/hudson" xmlns:f="/lib/form"
          xmlns:c="/lib/credentials">
 
+    <f:description>A unique name for this EC2 Fleet cloud</f:description>
+    <f:description>Warning: Cloud name is used to track agents belonging to this cloud. Please do not modify an existing cloud's name. See <a href="https://github.com/jenkinsci/ec2-fleet-plugin/issues/382">this issue</a> for details</f:description>
```

Comment:
Is this warning the only barrier in place from changing the cloud name? What would happen to existing agents if a cloud name is changed?

Reply:
Unfortunately yes. I meant to create another task to implement disallowing name change. This PR is already handling a lot, so didn't want to overload it.